spring-boot-modules/spring-boot-mvc-5/src/main/java/com/baeldung/problemdetails/web/SalesController.java

```diff
@@ -0,0 +1,40 @@
+package com.baeldung.problemdetails.web;
+
+import com.baeldung.problemdetails.model.InvalidInputException;
+import com.baeldung.problemdetails.model.OperationRequest;
+import com.baeldung.problemdetails.model.OperationResult;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("sales")
+public class SalesController {
+
+  @PostMapping("/calculate")
+  public ResponseEntity<OperationResult> calculate(
+      @Validated @RequestBody OperationRequest operationRequest) {
+    OperationResult operationResult = null;
```

Comment:
No need to initialize to null the variable.

Reply:
Done